modules/_labs/combobox/react/lib/Combobox.tsx

```diff
@@ -51,6 +51,10 @@ export interface ComboboxProps extends GrowthBehavior, React.HTMLAttributes<HTML
    * The id of the form field.
    */
   labelId?: string;
+  /**
+   * Toggle the use of ARIA 1.0 or ARIA 1.1
+   */
+  useLegacyAriaAttributes: boolean;
```

Comment:
I'm not sure what to call this prop. It defaults to `true`. 1.0 isn't strictly legacy and there are problems already with the 1.1 spec... What if there's a 1.2 spec and 1.1 support is simply skipped?

What about:
```ts
useAriaSpec: '1.0' | '1.1';
```
with a default to `'1.0'`?